commons/src/main/java/org/codice/ddf/admin/commons/requests/RequestUtils.java

```diff
@@ -120,58 +127,38 @@ public ProbeReport sendGetRequest(String url, String userName, String password)
      * @param content - Body of the post request
      * @return report
      */
-    public ProbeReport sendPostRequest(String url, String username, String password,
-            String contentType, String content) {
+    public ProbeReport sendPostRequest(String url, String username, String password, String contentType, String content,
+                                       SecurityManager securityManager) {
         Report connectReport = endpointIsReachable(url);
```

Comment:
Is this method needed anymore? It looks like we will get the same result if we fail to generate the SecureCXF client.